docs/hosting/server-setups/digital-ocean.md

```diff
@@ -1,24 +1,37 @@
 # Hosting n8n on DigitalOcean
 
-This hosting guide shows you how to self-host n8n on a DigitalOcean droplet. It uses n8n and [Caddy](http://caddyserver.com){:target="_blank" class=.external-link} (a reverse proxy) to allow access to the Droplet from the internet. It uses [Docker Compose](https://docs.docker.com/compose/){:target="_blank"} to create and define the application components and how they work together.
+This hosting guide shows you how to self-host n8n on a DigitalOcean droplet. It uses:
+
+* [Caddy](http://caddyserver.com){:target="_blank" .external-link} (a reverse proxy) to allow access to the Droplet from the internet. 
+* [Docker Compose](https://docs.docker.com/compose/){:target="_blank" .external-link} to create and define the application components and how they work together.
+* The [n8n Docker image](https://hub.docker.com/r/n8nio/n8n){:target=_blank .external-link}.
```

Comment:
Hmm. If you really want to mention this, then really you should probably also mention that it uses the Caddy one, but personally, I'm not sure if this is really necessary to say as using Docker would imply that you're using images for the services anyway. Maybe a compromise might be to say in the line above that the Docker compose file uses those images, or actually would make more sense to mention that when discussing the contents of the Docker compose file I think.